src/main/java/cinemas/controllers/ApiController.java

```diff
@@ -0,0 +1,46 @@
+package cinemas.controllers;
+
+import cinemas.dtos.CityDto;
+import cinemas.dtos.TheaterDto;
+import cinemas.services.CityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+    @Autowired
+    private CityService cityService;
+
+
+    @RequestMapping(value = "/cities", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
    @RequestMapping(value = "/", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```
cơ mà em dùng GetRequest đi, để khỏi khai báo method nè